.changeset/neat-boxes-fail.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Looking at this changeset's generated filename, there's a joke about front-end development somewhere in there 😅